i3-agenda.py

```diff
@@ -21,6 +21,8 @@
                    help='path to your credentials.json file')
 parser.add_argument('--cachettl', '-ttl', type=int, default=30,
                    help='time for cache to be kept in minutes')
+parser.add_argument('--local-time', '-l', help='return events in local time',
```

Comment:
I think the help message should better reflect the change to the user than the developer, does this mean that the user will see the event time in local time?

Reply:
Yeah, good question.

Personally I think the local time should be returned by default (or enforced in this case?). I'm wondering though why the conversion didn't work as expected from the get-go, as in why Google calendar isn't returning it correctly. I'm going to investigate it a bit further and get back to you.

Reply:
Great, thanks :smile:

Reply:
Now I see the problem! :smile: 

```js
'start': {'dateTime': '2020-04-14T20:00:00Z', 'timeZone': 'Europe/Stockholm'}, 'end': {'dateTime': '2020-04-14T21:00:00Z', 'timeZone': 'Europe/Stockholm'}
```

It must be converted using the `timeZone` argument that is returned.

I can fix it later tonight and update this merge request. Then I guess it will work as it was supposed to work, right?

Reply:
If I understand correctly, the 00Z should tell it that it's +00, so I'm not sure if the timeZone field is necessary